proto/bragi/common.proto

```diff
@@ -59,6 +59,10 @@ message Match {
   uint32 away_score = 7;
   MatchStatusType match_status = 8;
   google.protobuf.Timestamp plannet_start = 9;
+  string sport_name = 10;
```

Comment:
Isn't it redundant? You already have: `Sport sport = 3;`

Reply:
Removed